### DISCUSSION TITLE
List of public functions in Sablier flow

### DISCUSSION THREAD

**Discussion by smol-ninja** [sablier-labs/flow#97] [Show and tell]
As discussed, the Flow contract will have the following non-view public functions: 

- `adjustRatePerSecond`
- `batch`
- `create`
- `createAndDeposit`
- `createAndDepositViaBroker`
- `deposit`
- `depositViaBroker`
- `pause`
- `depositAndPause`
- `refund`
- `refundAndPause`
- `restart`
- `restartAndDeposit`
- `withdrawAt`
- `withdrawMax`

Related discussion: https://github.com/sablier-labs/v2-open-ended/issues/92. FYI cc @sablier-labs/engineers. Please object if you have any comments.

**Comment by PaulRBerg** [comment]
LGTM but I will review again when I give the whole repo a final, proper review.
